apps/api/src/lib/coscrad-query-language/coscrad-query-language.spec.ts

```diff
@@ -398,7 +411,81 @@ describe(`Coscrad Query Language`, () => {
                     });
 
                     describe(`when 2 parameters are provided`, () => {
-                        it.todo(`should have a test once language code is supported`);
+                        describe(`when the search text is not empty`, () => {
```

Comment:
It would be good to have a test helper
```ts
const assertParameterError = (query: ICoscradUserQuery,checkError: (result) => void) =>{
// ..
}

const checkErrorMessage = (error: Error, expectedSnippets: string[]) =>{
// assert that each of the snippets is contained in the message
}
```